tests/tests_integration/test_api/test_simulators/conftest.py

```diff
@@ -47,3 +70,46 @@ def seed_simulator_integration(cognite_client: CogniteClient, seed_simulator, se
             "/simulators/integrations/update",
             json={"items": [{"id": integration.id, "update": {"heartbeat": {"set": int(time.time() * 1000)}}}]},
         )
+
+
+@pytest.fixture
+def seed_simulator_models(cognite_client: CogniteClient, seed_simulator_integration, seed_resource_names) -> None:
+    model_unique_external_id = seed_resource_names["simulator_model_external_id"]
+    models = cognite_client.simulators.models.list(limit=None)
+    model_exists = any(model.external_id == model_unique_external_id for model in models)
+
+    if model_exists:
+        return
+
+    # Only create if model doesn't exist
+    simulator_model["dataSetId"] = seed_resource_names["simulator_test_data_set_id"]
+    cognite_client.simulators.models._post(
+        "/simulators/models", json={"items": [{**simulator_model, "externalId": model_unique_external_id}]}
+    )
+
+
+@pytest.fixture
+def seed_simulator_model_revisions(cognite_client: CogniteClient, seed_simulator_models, seed_file) -> None:
+    model_unique_external_id = resource_names["simulator_model_external_id"]
+    model_revision_unique_external_id = resource_names["simulator_model_revision_external_id"]
+    model_revisions = cognite_client.simulators.models.revisions.list(
+        filter=SimulatorModelRevisionsFilter(model_external_ids=[model_unique_external_id])
+    )
+    model_revision_not_exists = (
+        len(list(filter(lambda x: x.external_id == model_revision_unique_external_id, model_revisions))) == 0
+    )
+
+    if model_revision_not_exists:
+        cognite_client.post(
+            f"/api/v1/projects/{cognite_client.config.project}/simulators/models/revisions",
+            json={
+                "items": [
+                    {
+                        **simulator_model_revision,
+                        "fileId": seed_file.id,
+                        "modelExternalId": model_unique_external_id,
+                        "externalId": model_revision_unique_external_id,
```

Comment:
```suggestion
                        "description": "test sim model revision description",,
                        "fileId": seed_file.id,
                        "modelExternalId": model_unique_external_id,
                        "externalId": model_revision_unique_external_id,
```

here is the definition of `simulator_model_revision`
```
simulator_model_revision = {
    "externalId": resource_names["simulator_model_revision_external_id"],
    "modelExternalId": resource_names["simulator_model_external_id"],
    "description": "test sim model revision description",
    "fileId": 00000000000000,
}
```

3 out of 4 params are overridden, therefore no need of it, I presume